### DISCUSSION TITLE
https://w3c.github.io/ vs. https://www.w3.org/

### DISCUSSION THREAD

**Discussion by rdebeasi** [w3c/wcag#3805] [Q&A] [answered]
Hello! I noticed that the WCAG Understanding docs appear on both https://w3c.github.io/wcag/understanding and https://www.w3.org/WAI/WCAG21/Understanding.

For example:
- https://w3c.github.io/wcag/understanding/timing-adjustable.html
  - Includes guidance on toast messages from #813
- https://www.w3.org/WAI/WCAG21/Understanding/timing-adjustable.html
  - Does not include guidance on toast messages

What is the difference between these two versions of the site? Is one considered "official" and the other a "draft", or is the GitHub version just updated more frequently? Thank you!

**Comment by patrickhlauke** [comment, marked as answer]
the `.github.io` is the rendered version of what's in the github repo, which always relates to the latest version working draft/version.

`w3.org` is the official published version.

for reference, it's not surprising in your case that the two differ (in this case, with the toast guidance), since the `.io` version now reflects WCAG 2.2 https://www.w3.org/WAI/WCAG22/Understanding/timing-adjustable.html (which does include the toast guidance)

**Reply by rdebeasi** [reply]
Cool, that's good to know. Thank you!!
